### PR TITLE
fix: cow non-payable protocol fees

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

tl;dr google docs with instructions was off - we were told to add protocol fees in token on top of the sell amount in https://docs.google.com/document/d/1w1jxRUfSbQU-15FgC-sw85pY2Ep8-VDACEwzvndylvk 
but the CoW API says otherwise - it's handling deducting these perfectly fine, thus `requiresBalance` should be false for everything to work fine including max token swaps.

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes https://github.com/shapeshift/web/issues/6891

Low - isolated to CoW

> What protocols, transaction types or contract interactions might be affected by this PR?

CoW

## Testing

- Non-max CoW swaps are still happy
- Max CoW swaps are still happy

### Engineering

- ^

### Operations

- ^

## Screenshots (if applicable)

https://jam.dev/c/3a5349df-d5cd-402b-b3d7-7f1ad9f175ee

### CODE REVIEW

File: src/lib/swapper/swappers/CowSwapper/getCowSwapTradeQuote/getCowSwapTradeQuote.ts [hunk 1]
```diff
@@ -133,7 +133,8 @@ export async function getCowSwapTradeQuote(
           protocolFees: {
             [sellAsset.assetId]: {
               amountCryptoBaseUnit: feeAmountInSellTokenCryptoBaseUnit,
-              requiresBalance: true,
+              // Technically does, but we deduct it off the sell amount
```
Comment (woodenfurniture) [inline src/lib/swapper/swappers/CowSwapper/getCowSwapTradeQuote/getCowSwapTradeQuote.ts:136]: ❤️ comments are good.

File: src/state/apis/swapper/helpers/validateTradeQuote.ts [hunk 2]
```diff
@@ -157,17 +153,8 @@ export const validateTradeQuote = async (
...
-  // CowSwap fees used to be paid directly at execution-time, but they now need to be included in the order in *addition* to the amount being traded
-  const networkFeeRequiresBalance = true
-  const firstHopPayableProtocolFeeCryptoPrecision =
-    swapperName === SwapperName.CowSwap
-      ? fromBaseUnit(
-          bnOrZero(firstHop.feeData.protocolFees[firstHop.sellAsset.assetId]?.amountCryptoBaseUnit),
-          firstHop.sellAsset.precision,
-        )
-      : bn(0).toFixed()
+  // Technically does, but we deduct it off the sell amount
```
Comment (woodenfurniture) [inline src/state/apis/swapper/helpers/validateTradeQuote.ts:156]:
```suggestion
  // Technically does for cow swap too, but we deduct it off the sell amount in that case
```

**Review by woodenfurniture — APPROVED** [review]
